cmd/fleetctl/testdata/expectedHostDetailResponseJson.json

```diff
@@ -1 +1 @@
-{"kind":"host","apiVersion":"v1","spec":{"created_at":"0001-01-01T00:00:00Z","updated_at":"0001-01-01T00:00:00Z","id":0,"detail_updated_at":"0001-01-01T00:00:00Z","label_updated_at":"0001-01-01T00:00:00Z","policy_updated_at":"0001-01-01T00:00:00Z","last_enrolled_at":"0001-01-01T00:00:00Z","seen_time":"0001-01-01T00:00:00Z","refetch_requested":false,"hostname":"test_host","uuid":"","platform":"","osquery_version":"","os_version":"","build":"","platform_like":"","code_name":"","uptime":0,"memory":0,"cpu_type":"","cpu_subtype":"","cpu_brand":"","cpu_physical_cores":0,"cpu_logical_cores":0,"hardware_vendor":"","hardware_model":"","hardware_version":"","hardware_serial":"","computer_name":"test_host","primary_ip":"","primary_mac":"","distributed_interval":0,"config_tls_refresh":0,"logger_tls_period":0,"team_id":null,"pack_stats":null,"team_name":null,"gigs_disk_space_available":0,"percent_disk_space_available":0,"issues":{"total_issues_count":0,"failing_policies_count":0},"labels":[],"packs":[],"policies":[{"id":1,"query_id":2,"query_name":"query1","query_description":"","response":"passes","resolution":""},{"id":2,"query_id":43,"query_name":"query2","query_description":"","response":"fails","resolution":""}],"status":"mia","display_text":"test_host"}}
```

Comment:
We should figure out a way to store these jsons in a more readable way, so also the diffs make sense when a test fails... we need to improve this. Adding to the tech debt list.

Reply:
For posterity: #3047.